test/withdrawUnitTest.js

```diff
@@ -44,23 +44,23 @@ contract('ROSCA withdraw Unit Test', function(accounts) {
 
     it("Watches for event LogFundsWithdrawal()", co(function* () {
       for (let roscaHelper of [ethRoscaHelper, erc20RoscaHelper]) {
-        const ACTUAL_CONTRIBUTION = consts.CONTRIBUTION_SIZE * 0.8;
+        const ACTUAL_CONTRIBUTION = consts.CONTRIBUTION_SIZE * 1.8;
+        const EXPECTED_WITHDRAWAL = consts.CONTRIBUTION_SIZE * 0.8;
 
         yield roscaHelper.contribute(0, ACTUAL_CONTRIBUTION);
 
         let result = yield roscaHelper.withdraw(0);
         let log = result.logs[0];
 
-        assert.equal(log.args.user, accounts[0], "LogContributionMade doesn't display proper user value");
```

Comment:
we actually call contribute. Should this assert pass?

Reply:
We are only watching for LogFundsWithdrawal, it was an error on previous version. only the error string was wrong. the assert is right